### PR TITLE
fees and payments

### PR DESCRIPTION
**Pull request by mrganeshraja**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/middleware/updateApplicationFeeMiddleware.js [hunk 1]
```diff
@@ -35,13 +38,29 @@ const getFeeFromService = () => {
     });
 };
 
+const applicationFeeAndPaymentsQueryParams = '/fees-and-payments/version/1/petition-issue-fee';
```
Comment (tchow8) [inline app/middleware/updateApplicationFeeMiddleware.js]: May not work when postMapping is @PostMapping("/version/1/fee") ?

File: app/middleware/updateApplicationFeeMiddleware.js [hunk 2]
```diff
@@ -35,22 +39,48 @@ const getFeeFromService = () => {
...
+  //const service = CONF.deployment_env === 'local' ? mockFeesAndPaymentsService : feesAndPaymentsRegisterService;
+  const service = CONF.deployment_env === 'local' ? feesAndPaymentsRegisterService : feesAndPaymentsRegisterService;
+  
+
+  return service.get()
+    .then(response => {
+      // set fee returned from fee register to global CONF
+      logger.info(" Fee code set to ", response.feeCode);
+      CONF.commonProps.code = response.feeCode;
+      logger.info(" Fee code set to ", response.version);
```
Comment (tchow8) [inline app/middleware/updateApplicationFeeMiddleware.js]: Can change the message to Fee version set to :)

File: app/middleware/updateApplicationFeeMiddleware.js [hunk 3]
```diff
@@ -35,22 +39,48 @@ const getFeeFromService = () => {
     });
 };
 
+const getFeeCodeFromFeesAndPayments = () => {
+  //const service = CONF.deployment_env === 'local' ? mockFeesAndPaymentsService : feesAndPaymentsRegisterService;
+  const service = CONF.deployment_env === 'local' ? feesAndPaymentsRegisterService : feesAndPaymentsRegisterService;
```
Comment (tchow8) [inline app/middleware/updateApplicationFeeMiddleware.js]: Probably intended for the commented out line?

**Comment by nsingamsetty** [comment]
This is verified with Tony and see the Fee Code and Version in Logs. Performed an E2E test and looking good.PR build is successful.
https://pr-210-div-pfe-preview.service.core-compute-preview.internal/index.

**Pull request merged**